spec/models/library_entry_spec.rb

```diff
@@ -134,6 +135,49 @@
     end
   end
 
+  describe 'timestamp validation' do
+    let!(:anime) { create(:anime) }
```

Comment:
it will create a new object each time in the `before(:each)` and the way to reference it is `Anime.last`. If you need to keep referencing the same object you can do `before do` so it will only do it once.